docs/UserGuide.md

```diff
@@ -196,7 +196,7 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+  ![result for 'find alex david'](images/findJaneTanResult.png)
```

Comment:
I feel it is better to include the command as well. But no worries, I am going to fix it.